src/models/jira/IssueFields.gs

```diff
@@ -339,16 +341,19 @@ IssueFields = (function () {
       if (jiraFieldResponse.schema.custom.indexOf(':gh-epic-label') > -1) {
         EpicField.setLabelKey(jiraFieldResponse.key || jiraFieldResponse.id);
       }
+      customType = jiraFieldResponse.schema.custom;
```

Comment:
Recording this field so we can differentiate sprint fields and handle them differently